demo/AngularApp/references.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference path="./node_modules/tns-platform-declarations/ios.d.ts" />
```

Comment:
Because of [native](https://github.com/NativeScript/nativescript-dev-webpack/pull/578/files#diff-53c17be1e153dad7eee9058fa5257984R4) API - otherwise, transpilation fails.

Reply:
cast to any? I mean, it's a test app.